doc/install/check_installation.rst

```diff
@@ -40,8 +40,8 @@ MNE-Python and its dependencies. Typical output looks like this::
     qtpy:          2.0.1 {PySide6=6.2.4}
 
 
-.. collapse:: |hand-paper| If you get an error...
-    :class: danger
+.. dropdown:: If you get an error...
+    :icon: alert-fill
```

Comment:
here (and in other dropdowns) we can also later add things like `:color: warning` or `:color: danger`, but at present they don't work yet; see https://github.com/executablebooks/sphinx-design/issues/70